state/migrations/offerconntections_mock_test.go

```diff
@@ -8,7 +8,7 @@ import (
 	reflect "reflect"
 
 	gomock "github.com/golang/mock/gomock"
-	v3 "github.com/juju/description/v3"
+	v3 "github.com/juju/description/v4"
```

Comment:
alias should be changed to v4

Reply:
I think the mocks just need regenerating. In fact, that may even change them to the more sensible `description`